sycl/include/CL/sycl/INTEL/esimd/esimd_view.hpp

```diff
@@ -389,7 +429,7 @@ template <typename BaseTy, typename RegionTy> class simd_view {
   // - std::pair<top_region_type, base_region_type>
   //
   RegionTy M_region;
-};
+}; // namespace gpu
```

Comment:
Missed this one in the initial review. Please fix.
```suggestion
};
```

Reply:
I will work on some change to the same file again. Could I fix it in the next change. So I can move on working on the next change

Reply:
@kbobrovs, fix it. Please review and merge timely.